src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -180,6 +180,8 @@ public TracerSettings(IConfigurationSource source)
 
             TraceBatchInterval = source?.GetInt32(ConfigurationKeys.SerializationBatchInterval)
                         ?? 100;
+
+            AspnetRouteTemplateResourceNamesEnabled = IsFeatureFlagEnabled(ConfigurationKeys.FeatureFlags.AspnetRouteTemplateResourceNamesEnabled);
```

Comment:
Most tracer settings can be set in several ways: environment variables, `app.config`/`web.config` file, or in a json file (`~/datadog.json` by default). Most exceptions to this rule are settings used in native code (env vars only).

Currently, `DD_TRACE_ASPNET_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED` will only work as an env var. Was this limitation intentional?

Reply:
Oops, no, I thought I'd reverted that change (turns out that was my other PR https://github.com/DataDog/dd-trace-dotnet/pull/1289). Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1288/commits/4b0e7b1ac49093fe57bdd001c90d2bb783dca545